Source/Laser/LaserParticleContainer.cpp

```diff
@@ -52,7 +52,21 @@ LaserParticleContainer::LaserParticleContainer (AmrCore* amr_core, int ispecies,
 
     pp.query("pusher_algo", m_pusher_algo);
     getWithParser(pp, "wavelength", m_wavelength);
-    getWithParser(pp, "e_max", m_e_max);
+    const bool e_max_is_specified = queryWithParser(pp, "e_max", m_e_max);
+    Real a0;
+    const bool a0_is_specified = queryWithParser(pp, "a0", a0);
+    if (a0_is_specified){
+        Real omega = 2*MathConst::pi*PhysConst::c/m_wavelength;
+        m_e_max = PhysConst::m_e * omega * PhysConst::c * a0 / PhysConst::q_e;
```

Comment:
I think this is correct, but the equivalent formula in `parameters.rst` for `e_max` is lacking a `^2` on the `c`.

Can you please also add the new `a0` option? :)